wordpress-mobile/WordPress-iOS
# [Stats Refresh] Add Period > Search Terms card

Fixes #10887 

To test:

---
- On a site with search terms, go to Stats > a period view.
- Verify the view is:
<img width="350" alt="search_terms" src="https://user-images.githubusercontent.com/1816888/51698497-9cbb8a00-1fc7-11e9-9e56-69bb3d225b2a.png">

---
- On a site with _no_ search terms, go to Stats > a period view.
- Verify the view is:
<img width="350" alt="no_search_terms" src="https://user-images.githubusercontent.com/1816888/51698528-b2c94a80-1fc7-11e9-8cbe-9f8c196ee945.png">
